app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def create
+    @user = User.find_by(id: params[:followed_id])
+    current_user.follow(@user)
```

Comment:
```suggestion
    current_user.follow @user
```
làm tương tự fix những chỗ khác